src/expr.cpp

```diff
@@ -7299,8 +7355,30 @@ llvm::Constant *AddressOfExpr::GetConstant(const Type *type) const {
     if (ft != NULL) {
         llvm::Constant *c = expr->GetConstant(ft);
         return lConvertPointerConstant(c, type);
-    } else
-        return NULL;
+    } else {
```

Comment:
Removing "else" branch here, it would save 1 level of nesting.

Reply:
done